site/templates/page/homepage.jet

```diff
@@ -40,4 +40,14 @@
     {{end}}
 
   </div>
+
+  <script>
+    document.querySelectorAll('.btn-trailer').forEach(function(btn) {
```

Comment:
I think this code deserves to be in the main.js file.
It should also only fire once the dom has loaded.